python/ribasim/ribasim/utils.py

```diff
@@ -66,12 +66,6 @@ def _link_lookup(uds) -> Series[Int32]:
     )
 
 
-def _time_in_ns(df) -> None:
-    """Convert the time column to datetime64[ns] dtype."""
-    # datetime64[ms] gives trouble; https://github.com/pydata/xarray/issues/6318
-    df["time"] = df["time"].astype("datetime64[ns]")
```

Comment:
The linked issue is now fixed in xarray. I don't think we need to keep it to support old versions of optional dependencies.